frontend/src/v5/ui/routes/dashboard/projects/drawings/drawingDialogs/createDrawingDialog.component.tsx

```diff
@@ -59,38 +50,7 @@ export const CreateDrawingDialog = ({ open, onClickClose }) => {
 			maxWidth="sm"
 			{...formState}
 		>
-			<FormTextField
-				control={control}
-				name="name"
-				label={formatMessage({ id: 'drawings.creation.form.name', defaultMessage: 'Name' })}
-				formError={errors.name}
-				required
-			/>
-
-			<FormTextField
-				control={control}
-				name="number"
-				label={formatMessage({ id: 'drawings.creation.form.number', defaultMessage: 'Number' })}
-				formError={errors.number}
-				required
-			/>
-			<FormSelect
-				required
-				control={control}
-				label={formatMessage({ id: 'drawings.creation.form.type', defaultMessage: 'Category' })}
-				name="type"
-			>
-				{types.map((type) => (
-					<MenuItem key={type} value={type}> {type}</MenuItem>
-				))}
-			</FormSelect>
-			<FormTextField
-				control={control}
-				name="desc"
-				label={formatMessage({ id: 'drawings.creation.form.description', defaultMessage: 'Description' })}
-				formError={errors.desc}
-			/>
-			<UnhandledErrorInterceptor expectedErrorValidators={[nameAlreadyExists, numberAlreadyExists]} />
+			<DrawingForm formData={formData} />
```

Comment:
Calibration doesnt seem to work the first time.
- after calibrating the first time (Without refreshing the page) the camera and pins and not showing 
- If you enter to recalibrate , the saved 3d and 2d vectors are not showing

![calibration-doesntwork](https://github.com/user-attachments/assets/6b28a507-8ccb-407c-8485-6d58b051c816)